barretenberg/cpp/src/barretenberg/commitment_schemes/pcs_test_flavors.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "barretenberg/ecc/curves/bn254/bn254.hpp"
+#include "barretenberg/transcript/transcript.hpp"
+#include "commitment_key.test.hpp"
+
+namespace bb {
```

Comment:
We are already in commitment_schemes folder so this file and the other file don't need to be named pcs_*. I would also move them in `utils` since there are also other test related stuff there. These two Flavors could be  called Settings and be made `structs` since they're just data

Reply:
also will you please make use of these in the other pcs files as well (either in this PR or a follow up PR)